docs/core/diagnostics/cli-tools/dotnet-dump.md

```diff
@@ -0,0 +1,141 @@
+---
+title: dotnet-dump - .NET Core
+description: Installing and using the dotnet-dump command-line tool.
+author: sdmaclea
+ms.author: stmaclea
+ms.date: 08/21/2019
+---
+# Dump collection and analysis utility (`dotnet-dump`)
+
+The `dotnet-dump` CLI global tool is a way to collect and analyze Windows and Linux dumps all without any native debugger involved like `lldb` on Linux. This tool is important on platforms like Alpine Linux where a fully working `lldb` isn't available. The `dotnet-dump` tool will allow you to run SOS commands to analyze crashes and the GC, but it isn't a native debugger so things like displaying native stack frames aren't supported.
+
+> [!NOTE]
+> `dotnet-dump` is not supported on macOS.
+
+## Installing `dotnet-dump`
+
+To install the latest release version of the `dotnet-dump` [NuGet package](https://www.nuget.org/packages/dotnet-dump):
+
+```bash
```

Comment:
We usually use console and we'll have a special dotnetcli identifier soon.

```suggestion
```console
```